x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/utils.ts

```diff
@@ -73,3 +73,14 @@ export const calculateName = ({
     return 'untitled';
   }
 };
+
+/*
+  Alert throttle has to be null in case there are no actions or actions are exectured per rule exectution
```

Comment:
Thanks for the docs, but I would fix the spelling here and ensure you have a spelling checker like so:

<img width="789" alt="Screen Shot 2020-03-10 at 9 55 14 AM" src="https://user-images.githubusercontent.com/1151048/76333290-09badd80-62b7-11ea-8089-c21e503a986f.png">